app/assets/stylesheets/modules/request.scss

```diff
@@ -52,3 +52,7 @@ table.changed {
   cursor: not-allowed;
   opacity: 1;
 }
+
+.form-control:disabled { // Not sure why setting $input-bg-disabled doesn't work for this
```

Comment:
Anybody have any ideas why this isn't working the normal bootstrap way?

Reply:
`$input-disabled-bg` or `$input-bg-disabled`?

Reply:
That was it 🙄

Reply:
I'm sure in the next major version it'll be `$bg-disabled-input`